test/pred-sets/ref/inv_hash.cmp

```diff
@@ -5,8 +5,8 @@ bits:18
 lda:0
 0 ngram:
 0 skip:
-options: --hash_seed 0 --quadratic :: --oaa 3 --link identity
-Checksum: 2231945539
+options: --oaa 3 --quadratic ::
```

Comment:
I'm assuming you've made defaults implicit here.

Reply:
Yes, when default values are used they are no longer serialized into the model

Reply:
Yes, when default values are used they are no longer serialized into the model